modules/data/data_reader.py

```diff
@@ -50,8 +51,15 @@ def read_md(file_name: str) -> str:
     text = text.replace("{bot_tag}", escape_markdown(config_map['bot_tag'], version=2))
     return text
 
+reaction_path = get_abs_path("data", "yaml", "reactions.yaml")
+settings_path = get_abs_path("config", "settings.yaml")
 
-with open(get_abs_path("config", "settings.yaml"), 'r', encoding="utf-8") as yaml_config:
+# Read the local configuration
+with open(settings_path, 'r', encoding="utf-8") as yaml_config:
     config_map = yaml.load(yaml_config, Loader=yaml.SafeLoader)
-with open(get_abs_path("data", "yaml", "reactions.yaml"), 'r', encoding="utf-8") as yaml_config:
+
+# Read the local reactions and create it if not present
+if not os.path.exists(reaction_path):
+    shutil.copyfile(f"{reaction_path}.dist", reaction_path)
```

Comment:
mhmm, in another project where I use the .dist config structure I use to:
- load the .dist configurations (DEFAULT)
- if the NOT .dist file exists, use it to OVERRIDE the configuration that we've just loaded

why? because if we introduce a new conf in yaml.dist in the current .yaml the new conf will be not there and it will run an error.

I do not want to implement the same system but, maybe, we could avoid to make the copy of the .yaml.dist file if the .yaml does not exist, but we can just use the .yaml.dist

what do you think?

Reply:
I like this suggestion, so I implemented it. The _settings.yaml_ file still need to be created, since the bot token is sensitive information and using the .dist file directly could lead to some unpleasant errors